analysis/src/CreateInterface.ml

```diff
@@ -250,11 +254,22 @@ let printSignature ~extractor ~signature =
       Buffer.add_string buf (indent ^ newItemStr ^ "\n");
       processSignature ~indent items
     | Sig_type (id, typeDecl, resStatus) :: items ->
-      let newItemStr =
-        sigItemToString
-          (Printtyp.tree_of_type_declaration id typeDecl resStatus)
+      let lines =
```

Comment:
This fix is similar to the existing `Sig_value` attribute handling:

https://github.com/rescript-lang/rescript/blob/a91a2772fa625af440f76bb96e194871d68f4143/analysis/src/CreateInterface.ml#L233-L247